src/overlays/actors/ovl_En_Ex_Ruppy/z_en_ex_ruppy.c

```diff
@@ -21,29 +34,345 @@ const ActorInit En_Ex_Ruppy_InitVars = {
     (ActorFunc)EnExRuppy_Update,
     (ActorFunc)EnExRuppy_Draw,
 };
-*/
+
+Vec3f D_80A0B358[] = {{0.0f, 0.1f, 0.0f}, {0.0f, 0.0f, 0.0f}};
+Vec3f D_80A0B370[] = {{0.0f, 0.01f, 0.0f},{0.0f, 0.0f, 0.0f}};
+Vec3f D_80A0B388 = {0.0f, 0.1f, 0.0f};
+Vec3f D_80A0B394 = {0.0f, 0.0f, 0.0f};
+static Vec3f sPoint1Vec = {0x0, 0.1f,0x0};
+static Vec3f sZeroVector = {0.0f, 0.0f, 0.0f};
```

Comment:
these would all be much better off as inline data. instead of making them static and up here at the top of the file, move it into the function. when you do this you wont have to make a temp stack variable for it 

for example instead of 

    Vec3f unusedVec3f;
    Vec3f unusedVec3f2;
    unusedVec3f = D_80A0B388;
    unusedVec3f2 = D_80A0B394;

you can just put 

Vec3f D_80A0B388 = {0.0f, 0.1f, 0.0f};
Vec3f D_80A0B394 = {0.0f, 0.0f, 0.0f};

in its place in the function.

also, you need to make the 0s in sPoint1Vec  a float